### PR TITLE
feat: add Loom

### PR DESCRIPTION
**Pull request by paulius005**
This PR adds https://www.loom.com/

desktop_screenshot

### CODE REVIEW

File: apps/loom/loom.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Loom
+description: 'Instantly shareable screen and camera recordings'
```
Comment (Sanjit1) [inline apps/loom/loom.yml]: Add a period before you close the quotes on `line 2`
Pretty common mistake.

Comment (paulius005) [reply apps/loom/loom.yml]: fixed

**Comment by paulius005** [comment]
proof: https://www.loom.com/share/4ed069eb1c2648d2b190954b6000dddc

**Comment by paulius005** [comment]
thanks @Sanjit1

**Pull request merged**